src/apps/core/models/entity.py

```diff
@@ -30,5 +30,19 @@ class Entity(AbstractBaseModel):
         blank=True,
     )
 
+    def __str__(self):
```

Comment:
use single_translation() function from /common/helpers.py:298 to get a correct translation for title and description.

Reply:
Done in latest commit